### ISSUE TITLE
Create an endpoint for profile

### ISSUE THREAD

**Issue by MucahitErdoganUnlu** [bounswe/bounswe2024group11#288]
# 🧱 Description

Create an update profile API endpoint such that if a registered user has logged in, he can post a request to update his biography on his profile.

**Comment by MucahitErdoganUnlu** [comment]
still no progress. will do it after milestone 2

**Comment by MucahitErdoganUnlu** [comment]
duplicate of #388
